requirements/edx/github.txt

```diff
@@ -58,7 +59,7 @@ git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-clie
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 -e git+https://github.com/edx/edx-organizations.git@release-2015-08-31#egg=edx-organizations
 
-git+https://github.com/edx/edx-proctoring.git@0.8.3#egg=edx-proctoring==0.8.3
+git+https://github.com/edx/edx-proctoring.git@044a63d701e1516dae7861e8964b2666825d54e0#egg=edx-proctoring==0.7.3
```

Comment:
@wedaly we should talk about this hash here. After this RC cutting we'll be up in the 0.9.x tagged version range. Maybe Friday afternoon we should talk about you rebasing your branch and what I should do with your corresponding PR in edx-proctoring...